samples/Rewrite/ConditionParser/ConditionRegexParser.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Rewrite.ConditionParser
+{
+    public static class ConditionRegexParser
```

Comment:
Renamed for less confusion. It isn't necessary a regex expression, can contain string or int comparison and bash-esque flags.